src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let curentState = { ...state };
```

Comment:
There is a typo in the variable name `curentState`. It should be `currentState`.